subsystems/drivebase/SwerveDrive.java

```diff
@@ -64,17 +69,24 @@ public abstract class SwerveDrive extends DriveBase {
     public static record Config(double kDriveBaseTrackWidth, double kDriveBaseWheelBase,
             double kMaxVelocity,double kMaxAngularVelocity, double kMaxVoltage, double kMaxDriveCurrent,
             double kMaxSteerCurrent, SerialPort.Port kNavXPort, GearRatio kSwerveModuleGearRatio,
+
             int kFrontLeftModuleDriveMotor, int kFrontLeftModuleSteerMotor,
+            Motor_Type kFrontLeftDriveMotorType, Motor_Type kFrontLeftSteerMotorType, 
```

Comment:
We already have a ton of things in this Config record. I think it's safe to assume we'll never use one motor type for the front right drive and a different motor type for the front left drive. Let's just set a steerMotorType and a driveMotorType and reduce the chance of making mistakes